packages/tasit-contracts/test/SimpleStorage.js

```diff
@@ -0,0 +1,15 @@
+const SimpleStorage = artifacts.require("./SimpleStorage.sol");
+
+contract("SimpleStorage", function(accounts) {
+  it("should get the value", async function() {
+    const instance = await SimpleStorage.deployed();
+
+    const value = await instance.getValue();
```

Comment:
It will probably be a point of confusion for new developers coming to this project that we'll have to get used to the web3.js API for writing truffle tests but then use the ethers.js API and our own API also in JavaScript for testing our own code. So the ethers.js tests in `tasit-act` will look different than the truffle tests in `tasit-contracts` testing the same contract.

Reply:
Yes, I agree. If it will be a big problem, we can have no test at this package and use it only for migrations.